darwin/importer/importer.py

```diff
@@ -267,6 +268,45 @@ def _resolve_annotation_classes(
     return local_classes_not_in_dataset, local_classes_not_in_team
 
 
+def build_list_properties(annotations: List[dt.Annotation]) -> List[SelectedProperty]:
+    return [prop for annotation in annotations for prop in annotation.properties or []]
```

Comment:
I think this might need to also return the annotation.class info alongside the SelectedProperty, in a similar vein to metadata.json. See slack for more info.